package.json

```diff
@@ -12,5 +12,8 @@
     "grpc-tools": "^1.10.0",
     "grpc_tools_node_protoc_ts": "^5.1.0",
     "protoc-gen-grpc-web": "^1.2.1"
+  },
```

Comment:
since we use buf, we do not need anymore pkg.json?

Reply:
yeah I was thinking the same, i just wanted confirmation before removing